### PR TITLE
Another user's profile

### PR DESCRIPTION
**Pull request by active-sludge**
Navigation item Profile button now navigates to /myprofile instead of /profile.

/profile will be used for another user's profile.
Added dummy user link on homepage.
Added user or profile if statement to reuse profile.html.
Add myprofile url.

### CODE REVIEW

**Comment by jajimentol** [comment]
Followers/Followees shall be displayed with names, not emails. @active-sludge

**Comment by active-sludge** [comment]
On the homepage, user links are hardcoded for demonstration purposes.
There need to be at least 3 users, in order for the links to work.

File: Project/Wikode/wikodeApp/urls.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
     path('registration/', views.registration, name='registration'),
     path('registrationRequests/', views.registrationRequests, name='registrationRequests'),
     path('userLogin/', views.userLogin, name='userLogin'),
-    path('profile/', views.profilePage, name='profilePage'),
+    path('profile/', views.myProfilePage, name='myProfilePage'),
+    path('profile/<int:pk>', views.getProfilePageOfOtherfUser, name='getProfilePageOfOtherfUser'),
```
Comment (semihsolmaz) [inline Project/Wikode/wikodeApp/urls.py:12]: getProfilePageOfOtherfUser typo in view.

File: Project/Wikode/wikodeApp/views.py [hunk 2]
```diff
@@ -281,7 +281,35 @@ def getArticles(request):
...
+        'user': user,
+    }
+
+    return render(request, 'wikodeApp/profilePage.html', context)
+
+
+## Renders the profilePage.html with the clicked user's id information as pk
+## Navigates to /profile/# url.
+@login_required
+def getProfilePageOfOtherfUser(request, pk):
```
Comment (semihsolmaz) [inline Project/Wikode/wikodeApp/views.py:300]: getProfilePageOfOtherfUser typo

**Review by semihsolmaz — CHANGES_REQUESTED** [review]
just a typo in view name and corresponding URL

**Review by semihsolmaz — APPROVED** [review]

**Pull request merged**